src/main/fc/rc_adjustments.c

```diff
@@ -197,6 +198,37 @@ static const adjustmentConfig_t defaultAdjustmentConfigs[ADJUSTMENT_FUNCTION_COU
     }
 };
 
+#ifdef OSD
+#if (FLASH_SIZE > 2048)
```

Comment:
Don't check `FLASH_SIZE` directly. Instead define new build flag `USE_OSD_ADJUSTMENTS` and set it appropriately in `target/common_fc_pre.h`

Reply:
done